docs/core/tools/telemetry.md

```diff
@@ -106,7 +106,7 @@ The Microsoft distribution of .NET Core is licensed with the [MICROSOFT .NET LIB
 
 [.NET NuGet packages](https://www.nuget.org/profiles/dotnetframework) use the same license but don't enable telemetry (see [Scope](#scope)).
 
-> 2. DATA. The software may collect information about you and your use of the software, and send that to Microsoft. Microsoft may use this information to improve our products and services. You can learn more about data collection and use in the help documentation and the privacy statement at http://go.microsoft.com/fwlink/?LinkId=528096. Your use of the software operates as your consent to these practices.
+> 2. DATA. The software may collect information about you and your use of the software, and send that to Microsoft. Microsoft may use this information to improve our products and services. You can learn more about data collection and use in the help documentation and the privacy statement at https://www.visualstudio.com/en-us/dn948229. Your use of the software operates as your consent to these practices.
```

Comment:
since this is the actual text from what shows on the screen, I'd keep the Fwlink in this case

Reply:
Do you want me to only replace values surrounded in paren?

Reply:
It's ok the way you're doing. There are some things that I can only tell by looking at the link.